galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/routes/CPSRoute.java

```diff
@@ -43,104 +41,36 @@ public abstract class CPSRoute extends BaseRoute {
 
     protected IFramework framework;
 
-    private static final String REDACTED_PROPERTY_VALUE = "********";
+    protected PropertyUtilities propertyUtility;
 
-    private static final Set<String> hiddenNamespaces = new HashSet<>();
-    static {
-        hiddenNamespaces.add("dss");
-    }
 
-    /**
-     * Some namespaces are able to be set, but cannot be queried.
-     *
-     * When they are queried, the values are redacted
-     */
-    private static final Set<String> secureNamespaces = new HashSet<>();
-    static {
-        secureNamespaces.add("secure");
-    }
 
     public CPSRoute(ResponseBuilder responseBuilder, String path , IFramework framework ) {
     super(responseBuilder, path);
     this.framework = framework;
-    }
-
-    protected boolean isHiddenNamespace(String namespace){
-        return hiddenNamespaces.contains(namespace);
-    }
-
-    protected String getNamespaceType(String namespace){
-        String type = NamespaceType.NORMAL.toString();
-        if (CPSRoute.isSecureNamespace(namespace)){
-            type= NamespaceType.SECURE.toString();
-        }
-        return type;
-    }
-    
-
-    public static boolean isSecureNamespace(String namespace){
-        return secureNamespaces.contains(namespace);
-    }
-
-    protected String getProtectedValue(String actualValue , String namespace) {
-        String protectedValue ;
-        if (secureNamespaces.contains(namespace)) {
-            // The namespace is protected, write-only, so should not be readable.
-            protectedValue = REDACTED_PROPERTY_VALUE;
-        } else {
-            protectedValue = actualValue ;
-        }
-        return protectedValue ;
+    this.propertyUtility = new PropertyUtilities(framework);
```

Comment:
Not sure why but creating an instance of a utility class looks weird, usually their public methods are static, which makes them a pain to unit test.

Might be worth a discussion on how we could break it down into separate classes?